content/03-types/03-document/01-what-are-document-dbs.mdx

```diff
@@ -0,0 +1,107 @@
+
+Document databases offer a distinctly different structure and experience than traditional relational databases. Relational databases store data in programmer-defined separate tables that may see a single object spread across several tables. This separation requires [join statements](/intro/database-glossary#join) to get to a desired return from the database. The document model stores all of an object’s information in a single instance in the database, and every object in the database can be starkly different than the next. This capability, in theory, removes the need for an [object-relational mapper (ORM)](/intro/database-glossary#orm) depending on the use case.
+
+<PrismaOutlinks>
+
+Prisma's [MongoDB connector](https://www.prisma.io/docs/concepts/database-connectors/mongodb) has recently been [promoted to general availability](https://www.prisma.io/blog/mongodb-general-availability-pixnun6mffmu)!  With this change, you can use the [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) to manage production MongoDB databases with confidence.
+
+Join us in celebrating this milestone on April 25-29 with our [virtual MongoDB Launch week](https://www.prisma.io/mongodblaunch).  There will be exclusive workshops, opportunities for Atlas credits, great swag, and much more!
+
+</PrismaOutlinks>
+
+## Documents 
+As discussed, documents are at the crux of any document database. Depending on the document database, documents encapsulate and encode data in JSON, XML, YAML, or a binary form like BSON.
+
+One of the attractive elements for developers to use the document model is its similarities to objects in programming languages. There is a familiarity with the structure or lack thereof when using documents. 
+
+The basic format of a document looks as follows:
+
+```
+	 {
+	   field1: value1,
+	   field2: value2,
+	   field3: value3,
+	   ...
+	   fieldN: valueN
+	}
+```
+
+Expanding on the basic syntax, a single document within a collection of authors could look like this:
+
+```
+	{
+		"ID": "001",
+		"Books": { 'Grey Bees', 'Death and the Penguin' },
+		"Author": "Andrey Kurkov"
+	}
+```    
+
+What is critical to notice is the ability to store multiple books within the `Books` field. In a relational database, this would not be possible. There would need to be an `Author` table and a `Book` table joined by a [key](/intro/database-glossary#foreign-key). This foreign key would most likely be something like `author.id` where every record in the `Book` table. We can visualize the difference in the following tables:
```

Comment:
"This foreign key would most likely be something like `author.id` where every record in the `Book` table."  I might not be parsing this correctly, but I had trouble interpreting this.